src/events.cpp

```diff
@@ -1435,6 +1438,39 @@ bool onSpellCheck(Player* player, const Spell* spell)
 	return scriptInterface.callFunction(2);
 }
 
+void onConnect(Player* player, std::string& msg)
+{
+	// Player:onConnect(msg)
+	if (playerHandlers.onConnect == -1) {
+		return;
+	}
+
+	if (!tfs::lua::reserveScriptEnv()) {
+		std::cout << "[Error - events::player::onConnect] Call stack overflow" << std::endl;
+		return;
+	}
```

Comment:
unrelated to this PR, but these four lines appear every time a Lua event is bound in the sources
at this point it deserves a macro